governance/meeting.md

```diff
@@ -9,6 +9,25 @@ modified: 2023-03-03
 Summary of what has been discussed at meetings.
 
 ## Monthly meeting
+2023-04-20 (12 participants)
+* Attendees department: ESDC; NRCAN
+* General:
+    * No Honorable mentions since the last monthly meeting :  
+	* GCweb major version 13 was released on April 18: Few more adjustments were made to the version V6 of the footer. Therefore, the veterans link needs to be updated according to the latest version of the footer if the free style template is used.
+	* The GC web version 13 could not be released in AEM because of the ongoing labour disruption.
```

Comment:
Juste pour rassurer le public, j'ajouterais ceci :
```suggestion
	* The GC web version 13 could not be released in AEM because of the ongoing labour disruption, however this has no impact nor side effect whatsoever on AEM authors and end users.
```